src/main/java/org/alliancegenome/curation_api/interfaces/AGRCurationSchemaVersion.java

```diff
@@ -1,10 +1,14 @@
 package org.alliancegenome.curation_api.interfaces;
 
-import java.lang.annotation.*;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
 
 @Retention(RetentionPolicy.RUNTIME)
 public @interface AGRCurationSchemaVersion {
 
-	String value();
```

Comment:
The only draw back to not having value is you can't specify the `AGRCurationSchemaVersion("default value");` it might be nice to have "default value" set both min and max.

Reply:
But it might be nice to do it on a class by class basis.